test/trafficGenerator/SpawnPatternModel.spec.js

```diff
@@ -85,6 +85,24 @@ ava('#altitude returns a random altitude rounded to the nearest 1,000ft', (t) =>
     t.true(typeof result === 'number');
 });
 
+ava('#id returns #_id', (t) => {
+    const model = new SpawnPatternModel(ARRIVAL_PATTERN_MOCK);
+    const expectedResult = 'some-value';
+    model._id = expectedResult;
+    const result = model.id;
+
+    t.true(result === expectedResult);
+});
+
+ava('#positionModel returns #_positionModel', (t) => {
+    const model = new SpawnPatternModel(ARRIVAL_PATTERN_MOCK);
+    const expectedResult = 'some-value';
+    model._positionModel = expectedResult;
+    const result = model.positionModel;
+
+    t.true(result === expectedResult);
+});
```

Comment:
@n8rzz your thoughts on this? I find it hard to see a real benefit.

Reply:
🤔 Hmm.  Yeah, I agree, not sure why we have this test cause it isn't really doing anything valuable.  Lets remove it.

Reply:
We can certainly remove it if you want. It's definitely pretty useless, but we have plenty of tests just like it for the sake of getting to 100% coverage-- that's the only reason I added it: to prevent coveralls from showing a coverage decrease.